packages/contracts/contracts/L1/interfaces/IAssetManager.sol

```diff
@@ -528,10 +459,9 @@ interface IAssetManager {
     function undelegateKghBatch(address validator, uint256[] calldata tokenIds) external;
 
     /**
-     * @notice Claim the reward of the KGH delegator from the given validator vault and token ids.
+     * @notice Claim the boosted reward of the KGH delegator from the given validator vault.
```

Comment:
Let's use `boost reward` instead of `boosted reward`. Steve commented this in the medium article.

Reply:
I think the difference between code and medium article is a trivial thing and we use boosted reward all the places in code, it's better to unify to boosted reward in code base. wdyt?

Reply:
Yeah, it's trivial, but I thought it would be good to unify the terms in our all resources.

Reply:
So you mean it's better to change all the `boosted` terms in the code to `boost`?

Reply:
Let's use `boosted` right now.